src/stacks-hierarchy/Root_PurchaseOverviewScreen/components/FareProductHeader.tsx

```diff
@@ -1,33 +1,78 @@
-import {getTextForLanguage, useTranslation} from '@atb/translations';
+import {
+  getTextForLanguage,
+  PurchaseOverviewTexts,
+  useTranslation,
+} from '@atb/translations';
 import {StyleProp, View, ViewStyle} from 'react-native';
-import {FareProductTypeConfig} from '@atb/configuration';
+import {
+  FareProductTypeConfig,
+  PreassignedFareProduct,
+} from '@atb/configuration';
 import React, {forwardRef} from 'react';
 import {ThemeText} from '@atb/components/text';
 import {StyleSheet} from '@atb/theme';
 import {TransportationIconBoxList} from '@atb/components/icon-box';
+import {Button} from '@atb/components/button';
+import {Info} from '@atb/assets/svg/mono-icons/status';
+import {useIsTicketInformationEnabled} from '@atb/stacks-hierarchy/Root_PurchaseOverviewScreen/use-is-ticket-information-enabled';
 
 type Props = {
   fareProductTypeConfig: FareProductTypeConfig;
+  preassignedFareProduct: PreassignedFareProduct;
+  onTicketInfoButtonPress: () => void;
   style?: StyleProp<ViewStyle>;
 };
 export const FareProductHeader = forwardRef<View, Props>(
-  ({fareProductTypeConfig, style}: Props, ref) => {
-    const {language} = useTranslation();
+  (
+    {
+      fareProductTypeConfig,
+      preassignedFareProduct,
+      onTicketInfoButtonPress,
+      style,
+    }: Props,
+    ref,
+  ) => {
+    const {t, language} = useTranslation();
     const styles = useStyle();
+    const [isTicketInformationEnabled] = useIsTicketInformationEnabled();
 
     return (
-      <View style={[style, styles.header]} ref={ref} accessible={true}>
-        <TransportationIconBoxList
-          modes={fareProductTypeConfig.transportModes}
-          iconSize="normal"
-        />
-        <ThemeText
-          type="heading--medium"
-          color="background_accent_0"
-          style={{flexShrink: 1}}
-        >
-          {getTextForLanguage(fareProductTypeConfig.name, language) ?? ''}
-        </ThemeText>
+      <View style={style}>
+        <View style={styles.header} ref={ref} accessible={true}>
+          <TransportationIconBoxList
+            modes={fareProductTypeConfig.transportModes}
+            iconSize="normal"
+          />
+          <ThemeText
+            type="heading--medium"
+            color="background_accent_0"
+            style={styles.headerText}
+          >
+            {getTextForLanguage(fareProductTypeConfig.name, language) ?? ''}
+          </ThemeText>
+        </View>
+        {isTicketInformationEnabled && (
+          <View style={styles.headerSubSection}>
+            <ThemeText
+              type="body__secondary"
+              color="background_accent_0"
+              style={styles.ticketDescription}
+              numberOfLines={1}
+            >
+              {getTextForLanguage(
+                preassignedFareProduct.productDescription ?? [],
+                language,
+              )?.replaceAll('\n', ' ')}
```

Comment:
Should it also look for `<br>`, `<br/>` and `<br />`?

Reply:
Ideally, I think all markup should be should be stripped: https://github.com/AtB-AS/mittatb-app/pull/4127#discussion_r1427975624

Reply:
Resolving this and moving discussion to https://github.com/AtB-AS/mittatb-app/pull/4127#discussion_r1428035977